### PR TITLE
GO-4017 Set imageKind on file upload

### PR DESCRIPTION
**Pull request by KirillSto**
https://linear.app/anytype/issue/GO-4017/clipboard-html-files-should-be-hidden-by-default

- **AutomaticallyCreated** image kind is introduced in model protocol
- **AutomaticallyCreated** is set to all images being uploaded on bookmark creation, paste from html and webclipper
- **ImageKind** field is added to **UnspashDownload**, **FileUpload** and **BlockFileCreateAndUpload** RPCs
- **UploadBlockFile**, **UploadBlockFileSync** and **UploadFileBlock** methods of **block.Service** are united into **UploadBlockFile**
- **MarkIsArchived** and **BlockDataviewRelationListAvailable** are removed as they are not used

UPD:
- **isHiddenDiscovery** flag is set to all file objects with imageKind=automaticallyCreated

### CODE REVIEW

File: core/block/import/common/syncer/types.go [hunk 1]
```diff
@@ -10,13 +10,14 @@ import (
...
 	"github.com/anyproto/anytype-heart/core/domain/objectorigin"
+	"github.com/anyproto/anytype-heart/core/session"
 )
 
 type BlockService interface {
 	GetObject(ctx context.Context, objectID string) (sb smartblock.SmartBlock, err error)
 	GetObjectByFullID(ctx context.Context, id domain.FullID) (sb smartblock.SmartBlock, err error)
 	UploadFile(ctx context.Context, spaceId string, req block.FileUploadRequest) (objectId string, details *types.Struct, err error)
-	UploadFileBlock(contextID string, req block.UploadRequest) (fileObjectId string, err error)
+	UploadBlockFile(ctx session.Context, req block.UploadRequest, groupID string, isSync bool) (fileObjectId string, err error)
```
[the remark this answers is not in the document]
Comment (KirillSto) [reply core/block/import/common/syncer/types.go:20]: Yeah, I agree with you! Let me fix that

[the remark this answers is not in the document]
Comment (KirillSto) [reply core/block/import/common/syncer/types.go:20]: Basically, I have decided to propagate session.ctx in couple of file uploading commands

**Review by fat-fellow — APPROVED** [review]

**Review by AnastasiaShemyakinskaya — APPROVED** [review]

**Review by AnastasiaShemyakinskaya — APPROVED** [review]

File: core/block/editor/clipboard/clipboard.go [hunk 2]
```diff
@@ -541,6 +543,21 @@ func (cb *clipboard) pasteFiles(ctx session.Context, req *pb.RpcBlockPasteReques
 	return blockIds, cb.Apply(s)
 }
 
+func getImageKind(filename string) model.ImageKind {
+	imageFormats := []string{"jpg", "jpeg", "png", "gif", "webp"}
```
Comment (deff7) [inline core/block/editor/clipboard/clipboard.go:547]: Why do you use these image formats? Looks like the magic values from somewhere

Comment (KirillSto) [reply core/block/editor/clipboard/clipboard.go]: There was a business logic error. We should not put autoAdded kind to images pasted via clipboard (not from html), so I removed this func :smiley:

**Review by deff7 — APPROVED** [review]

**Pull request merged**